src/app/admin/speciality/speciality-list/speciality-list.component.ts

```diff
@@ -1,42 +1,27 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, ViewChild } from '@angular/core';
 import { Speciality } from '.././../entity.interface';
 import { ApiService } from '../api.service';
 import { DialogConfirmComponent, DialogConfirmModel } from '../dialog-confirm/dialog-confirm.component';
-
-import { ViewChild, TemplateRef } from '@angular/core';
-import { MatPaginator } from '@angular/material/paginator';
-import { MatDialog, MatTableDataSource, MatTable, MatSnackBar, MatSnackBarConfig } from '@angular/material';
-import { FormGroup, FormControl, Validators } from '@angular/forms';
-
+import { DialogFormComponent } from '../dialog-form/dialog-form.component';
+import { MatDialog, MatTableDataSource, MatTable, MatPaginator } from '@angular/material';
 
 @Component({
   selector: 'app-speciality-list',
   templateUrl: './speciality-list.component.html',
   styleUrls: ['./speciality-list.component.scss']
 })
 
-
 export class SpecialityListComponent implements OnInit {
 
   result: any;
   public entity = 'Speciality';
```

Comment:
In case you use `public` you should use it for all properties. Or you can just skip it. In TS `public` is default access control modifier.